Source/WebCore/platform/Scrollbar.h

```diff
@@ -25,6 +25,7 @@
 
 #pragma once
 
+#include "RenderStyleConstants.h"
```

Comment:
I think this is a layering violation, WebCore/platform should not include stuff outside it.

Possibly move the ScrollbarWidth enum to somewhere in WebCore/platform ? maybe ScrollTypes.h and include ScrollTypes.h where needed.

Reply:
Ah okay that makes sense. That's where ScrollbarControlSize was so makes sense to move ScrollbarWidth in there.